src/App.js

```diff
@@ -5,33 +5,178 @@ import {
+                <SelectWrap>
+                  <Select name="currency" id="current-currency" value={currency} onChange={this.handleChangeCurrency} theme={theme}>
+                    {Object.keys(currencyList).map((currency) => {
+                      return <option value={currency} key={keyGen()}>{currencyList[currency].name}</option>
+                    })
+                    }
+                  </Select>
+                <SelectArrow>&#9207;</SelectArrow>
+                </SelectWrap>
+              </NavRightSelectContainer>
+              <Mode theme={theme}><img src={mode} alt="mode" onClick={this.handleThemeButtonClick}/></Mode>           
+            </NavRight>
+          </Nav>
+
+          <NavUnderContainer>
+            <NavUnder theme={theme}>
+              {!global && <div>Loading Global API...</div>}
+              {global && <div>
+                <NavUnderUl>
+                  <NavUnderLi>Coins {global.active_cryptocurrencies}</NavUnderLi>
+                  <NavUnderLi>Exchange {global.markets}</NavUnderLi>
+                  <NavUnderLi><NavBulletPoint>&#8226;</NavBulletPoint>{currencySymbol}{prettierNumber(Math.round(global.total_market_cap[currency]))}</NavUnderLi>
+                  <NavUnderLi>
+                  <NavBulletPoint>&#8226;</NavBulletPoint>{currencySymbol}{prettierNumber(Math.round(global.total_volume[currency]))}
+                    <PercentDisplay percent={Math.round(global.total_volume[currency] / global.total_market_cap[currency])}>
+                      <Circle percent={Math.round(global.total_volume[currency] / global.total_market_cap[currency])} />
+                    </PercentDisplay>
+                  </NavUnderLi>
+                  <NavUnderLi>
+                    <NavUnderImg marginRight="1">
+                      <img src="https://assets.coingecko.com/coins/images/1/thumb/bitcoin.png?1547033579" alt="btc thumb"/>
+                    </NavUnderImg>
+                    {Math.round(global.market_cap_percentage.btc)}%
+                    <PercentDisplay percent={Math.round(global.market_cap_percentage.btc)}>
+                      <Circle percent={Math.round(global.market_cap_percentage.btc)}/>
+                    </PercentDisplay>
+                  </NavUnderLi>
+                  <NavUnderLi>
+                    <NavUnderImg marginRight="-4">
+                      <img src="https://assets.coingecko.com/coins/images/279/thumb/ethereum.png?1595348880" alt="eth thumb"/>
+                    </NavUnderImg>
+                    {Math.round(global.market_cap_percentage.eth)}%
+                    <PercentDisplay percent={Math.round(global.market_cap_percentage.eth)}>
+                      <Circle percent={Math.round(global.market_cap_percentage.eth)}/>
+                    </PercentDisplay>
+                  </NavUnderLi>
+                </NavUnderUl>
+              </div>}
+            </NavUnder>
+          </NavUnderContainer>
```

Comment:
Hey mate, maybe this Nav thing, should be a component

Reply:
Fixed, Also Did for the NavUnder